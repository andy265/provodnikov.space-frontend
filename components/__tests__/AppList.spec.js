const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const AppList = require('~/components/AppList')

describe('AppList.vue', () => {
  it('has same HTML structure', () => {
    let wrapper = shallow(AppList, {
      slots: {
        default: '<div>item 1</div>'
      }
    })
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
