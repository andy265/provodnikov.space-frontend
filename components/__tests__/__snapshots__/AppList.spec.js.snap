// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppList.vue has same HTML structure 1`] = `
<div class="app-list">
    <div>item 1</div>
</div>
`;
